dubbo-config/dubbo-config-spring/src/test/java/org/apache/dubbo/config/spring/impl/DemoServiceAnnotationImpl.java

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This file, too.

Reply:
this demo shows a situation when a class implements more than tow intefaces and just need to expot one of them(not the first one),so it is necessary to specify the interfaceName

Reply:
Nope, this project should only keep the simplest demo, just like it does now.
If u think this demo is necessary, pls send pr to here:
https://github.com/dubbo/dubbo-samples

Reply:
Got it

Reply:
:)
pls send this demo to here:
https://github.com/dubbo/dubbo-samples

thx for your pr.

Reply:
ok,I'll do that.